### PR TITLE
Us5 km update

### PR DESCRIPTION
**Pull request by Oluane**

### CODE REVIEW

File: frontend/src/components/style/NavBar.scss [hunk 1]
```diff
@@ -37,7 +37,7 @@ nav {
     #kmUpdate {
       width: 25%;
       margin: 0;
-      justify-content: center;
+      // justify-content: center;
```
Comment (BlandineM) [inline frontend/src/components/style/NavBar.scss]: comment

File: frontend/src/reducer/reducer.js [hunk 2]
```diff
@@ -1,9 +1,44 @@
...
+const reducer = (state = initialState, action) => {
+  const newState = JSON.parse(JSON.stringify(state));
 
-  return newState;
+  switch (action.type) {
+    case "UPDATE_KM_COUNTER":
+      const kmToUpdate = newState.kmToUpdate;
+
+      switch (action.value) {
+        case "erase":
```
Comment (LoicBrassart) [inline frontend/src/reducer/reducer.js:11]: I think this is bugged: You could enter the case (`action.value==='erase'`) and *not* enter the if (`numOfKmUpdates<6`). In this specific case: you wouldn't do anything but would not leave the switch, leading to the execution of all other cases ! :scream: 
More seriously, just get the `break` out of the `if`, just as you did in the lower case ;-)

File: frontend/src/components/style/NavBar.scss (hunk 1 above)
Comment (Oluane) [reply frontend/src/components/style/NavBar.scss]: done ;)

Comment (LoicBrassart) [inline frontend/src/components/style/NavBar.scss]: I'd say even more: dead code ! é_è

File: frontend/src/components/KmUpdate.jsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { connect } from "react-redux";
+import "./style/KmUpdate.scss";
+
+//prettier-ignore
+const keyboard = [1,2,3,4,5,6,7,8,9,"*",0,<img src="./pictures/icons/Supp.svg" />];
+
+function KmUpdate(props) {
+  console.log(typeof keyboard[11]);
```
Comment (LoicBrassart) [inline frontend/src/components/KmUpdate.jsx]: No `console.log` on my `dev` branch !

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Minor fixes to do, it won't take too long

**Review by BlandineM — DISMISSED** [review]

File: frontend/src/components/KmUpdate.jsx (hunk 3 above)
Comment (Oluane) [reply frontend/src/components/KmUpdate.jsx]: Shame on me !

**Review by BlandineM — DISMISSED** [review]

**Review by laurentmahieu — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
Seems good to me !

**Pull request merged**